scim2-sdk-common/src/main/java/com/unboundid/scim2/common/exceptions/UnauthorizedException.java

```diff
@@ -20,51 +20,74 @@
 import com.unboundid.scim2.common.messages.ErrorResponse;
 
 /**
- * Signals an authorization failure from the service provider.
+ * This class represents a SCIM exception pertaining to the
+ * {@code HTTP 401 UNAUTHORIZED} error response code. This exception type should
+ * be thrown when a client attempts to access a resource or endpoint when they
+ * do not have the appropriate authorization or access rights to do so. This
```

Comment:
Generally, one associates "access rights" with 403/forbidden, unless one is trying to not provide too much information to the caller.

Reply:
Good call. I've updated both the 401 and 403 pages to highlight this.